funnel/templates/notifications/proposal_received_web.html.jinja2

```diff
@@ -11,9 +11,9 @@
     </p>
     <ol class="mui-list--aligned">
       {%- for proposal in view.fragments %}
-        <li>{% trans url=proposal.url_for(), proposal=proposal.title, actor=proposal.owner.pickername, age=proposal.datetime|age %}
-          <a href="{{ url }}">{{ proposal }}</a> by {{ actor }} <span class="mui--text-body2">{{ age }}</span>
-        {% endtrans %}</li>
+        <li>{% trans url=proposal.url_for(), proposal=proposal.title, actor=proposal.owner.pickername %}
+          <a href="{{ url }}">{{ proposal }}</a> by {{ actor }}{% endtrans %} <span class="mui--text-body2">{{ proposal.datetime|age }}</span>
+        </li>
```

Comment:
This is worse. `age` is a sentence fragment. The assumption that it can be tacked on to the end 
 only works in English. Move it back inside and remove the span tag, _or_ move it out of the statement, into an independent line. @vidya-ram @StephanieBr some design input required here